### DISCUSSION TITLE
How do I set the default download location for catalog books?

### DISCUSSION THREAD

**Discussion by winteriscariot** [johnfactotum/foliate#1246] [Q&A]
I have an OPDS catalog provided by Calibre on my home server, however every time I want to read a book it asks me where I want to download it.

Is it possible to configure this, perhaps via a config file? There doesn't appear to be an option for this in Foliate settings

thanks!

**Comment by johnfactotum** [comment]
Currently, no, it's not possible.

In v2 it downloads to Foliate's data directory by default, but there's no such option in v3.
